### PR TITLE
Create/update/fix endrod

### PR DESCRIPTION
**Pull request by inxomnyaa**
## Introduction

See https://github.com/pmmp/PocketMine-MP/pull/940
This is a fixed version, working without problems. Drops and BB are fixed and the issues related to it are solved.

Because git is stupid, this commit also updates the branch from master

### CODE REVIEW

File: src/pocketmine/PocketMine.php [hunk 1]
```diff
@@ -84,6 +84,12 @@ function dummy(){
 	 * Enjoy it as much as I did writing it. I don't want to do it again.
 	 */
 
+	if(!extension_loaded("phar")){
```
Comment (jasonwynn10) [inline src/pocketmine/PocketMine.php]: I'm certain this should not be apart of this PR

**Comment by L3ice** [comment]
https://github.com/pmmp/PocketMine-MP/commit/c726874581a1812932cbfb9f5f9d004a5c15fad0

**Comment by L3ice** [comment]
https://github.com/pmmp/PocketMine-MP/commit/0e718c25e9a83cdbdbb0960b3f3245348d1fe1f7

**Comment by thebigsmileXD** [comment]
Thanks for clearing up obvious stuff @L3ice

**Comment by thebigsmileXD** [comment]
👍

**Pull request closed without merging**